types/cannon/index.d.ts

```diff
@@ -845,6 +849,35 @@ declare module CANNON {
 
     }
 
+    export class Trimesh extends Shape {
```

Comment:
Minor:

I see some more properties in https://github.com/schteppe/cannon.js/blob/569730f94a1d9da47967a24fad0323ef7d5b4119/src/shapes/Trimesh.js. Maybe it would be worth adding all of them (`normals`, `aabb`, `edges`, `tree`)?

Reply:
maybe layer

Reply:
Sure thing, no worries, it's optional anyway :slightly_smiling_face: